spec/app/rails_7.1/config/credentials/test.yml.enc

```diff
@@ -0,0 +1 @@
+qU0cAsjfKz4lnoJPU36JuM7Yh3qm6B2YV7LhqJROKCu4AsOW0AfFY9FM+aTeRsZOIdIBHyyargCI1xmq3N5o4rdVZRXxWIt2PD93xZPlcMrlAb645m8hyni1cW4=--dPokgPsIyoKhzMzD--nb/fnuH1FaTNT5iF8J3TfQ==
```

Comment:
I tried looking at these values locally with

```sh
cd spec/app/rails_7.1 
RAILS_MASTER_KEY='0e29551e5c31acf7c769d64397af54e4' bundle exec rails credentials:edit
```

but I got an error

```
Editing config/credentials.yml.enc...
Couldn't decrypt config/credentials.yml.enc. Perhaps you passed the wrong key?
```

...but it seems to work on CI, so it's a little weird. Any idea how I can look at these values locally?

Reply:
@cjlarose just remove the single quotes